server/src/amyserver_tools/temporal_features.py

```
@@ -28,8 +27,8 @@ class MultiScaleTemporalExtractor:
         >>> features = extractor.extract_and_fuse(sequence)
         >>> print(features.shape)  # (time_steps, fused_features)
     """
-    
-    def __init__(self, scales: List[int] = None):
+
+    def __init__(self, scales: list[int] = None):
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)